### DISCUSSION TITLE
Dubbi sulla connessione di dispositivi Zigbee ad antenna coordinator

### DISCUSSION THREAD

**Discussion by lnrd00** [indomus/forum#2914] [ZigBee, Z-Wave, Thread, Matter]
Salve, avrei un paio di domande, e per entrambe farò riferimento allo screen sottostante dove è presente la mappa dei miei dispositivi zigbee (moooolto scarna)
<img width="985" alt="Schermata 2024-02-15 alle 18 14 20" src="https://github.com/indomus/forum/assets/154708323/9783472c-122e-4eea-bdda-b2f99befab72">
1a domanda: a cosa si riferisce la connessione multipla del Sonoff mini? Sono i valori del link in trasmissione e quello in ricezione?
2a domanda: entrambi i dispositivi sono estremamente vicini all'antenna coordinator, e anche ponendoli accanto all'antenna il valore della connessione non raggiunge mai 255, come mai?
Io ho anche flashato l'antenna coordinator, una Sonoff ZBDongle-E, con firmare RCP nella speranza che questo potesse aumentare le prestazioni di quest'ultima, e infatti se non ricordo quando vidi la mappa subito dopo l'aggiornamento mi ricordo che il sensore zigbee SNZB02, che al momento era l'unico connesso (il Sonoff mini era staccato dalla rete elettrica), presentava un valore di 255 mentre adesso è nuovamente sceso a praticamente la metà.

Sapete dirmi qualcosa a riguardo? Grazie in anticipo!

**Comment by KC-inDomus** [comment]
purtroppo la valutazione dell'LQI (il valore di qualità del segnale) è un dato particolarmente ballerino, al quale francamente non si può dare molto peso. sulla prima domanda sì, dato che è un componente che funge da router (da ripetitore automatico di segnale) possibilmente quello è un doppio parametro che indica il valore di ricezione come end-device e il secondo come router (ma non ne ho certezza, dovresti vedere sulla documentazione zigbee2mqtt).
